docker/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-# Configure gsutil authentication
+# Configure gcloud authentication
```

Comment:
Sort of surprised this is needed, it's writing to `boto.cfg` which sounds like it's for AWS. But no need to mess with that now I guess.